### ISSUE TITLE
doze mode and high priority

### ISSUE THREAD

**Issue by edujugon** [davibennun/laravel-push-notification#123]
Hello,
I've been facing some issues when I send notifications to my apps since they don't arrive at the devices are in doze mode.

After researching, I've find out that it could be solve sending the notification from the backend with the parameter priority to high. In theory it will wake up the device and finally, makes the notification arrives.

anyone has faced that issue? how have you solved it?

**Comment by pavanpatharde** [comment]
@edujugon  I am also looking for the same documentation. I want to set the time_to_live option dynamically.

**Comment by pavankumarkatakam** [comment]
Actually we need to set the time to live option in the configuration file. But there is one issue with that
Please check the issues in this project

**Comment by edujugon** [comment]
@pavankumarkatakam Yes, you're right. it's not an issue in this package. Also, could you add the priority option to the configuration file?
It would be great if with that step we solve the problem for notifications that never arrive to the devices in doze mode..
